### DISCUSSION TITLE
IF i use this compose, how do i cvonfigure my openvpn settings? (airvpn)

### DISCUSSION THREAD

**Discussion by kyrusdemnati** [haugene/docker-transmission-openvpn#2183] [Q&A] [answered]
version: '3.3'
services:
    transmission-openvpn:
        cap_add:
            - NET_ADMIN
        volumes:
            - '/your/storage/path/:/data'
        environment:
            - OPENVPN_PROVIDER=PIA
            - OPENVPN_CONFIG=france
            - OPENVPN_USERNAME=user
            - OPENVPN_PASSWORD=pass
            - LOCAL_NETWORK=192.168.0.0/16
        logging:
            driver: json-file
            options:
                max-size: 10m
        ports:
            - '9091:9091'
        image: haugene/transmission-openvpn

**Comment by TheHesster** [comment, marked as answer]
Great documentation listed on the wiki.
https://haugene.github.io/docker-transmission-openvpn/config-options/
